s/web/templates/index.scss

```diff
@@ -65,3 +69,13 @@ nub-editor {
 	margin: auto;
 	margin-top: 1em;
 }
+
+nub-gridboard {
+	margin: 1em auto;
+	font-size: 2em;
+}
```

Comment:
why should lookpad work make the gridboard larger in font-size?

Reply:
oops, my bad, probably messing around and forgot to remove it, or just an error from when I fixed the conflict during the interactive rebase, will be sure to be more thorough next time